### DISCUSSION TITLE
Expand Auto-generated Release Notes to include main commits

### DISCUSSION THREAD

**Discussion by CaMer0n** [community/community#14517] [Other Feature Feedback, Questions, & Ideas]
The current release notes auto-generator is a fantastic time saver!! :clap:  :clap:  :clap: 
However,  it only generates a list of the merged pull-requests, excluding the main issues that were closed by the core developers. 
Sorry if this has been mentioned before (or already is possible and I missed it?), but.... 

Why not also include issues that were fixed/closed by core developers, and then sort them by tag name?
Better to have too much text (that can be edited/removed) at release time, than too little? 

### Bugs
* Issue title which has a bug tag by @bruce in `#0001 `
* Another issue with a bug tag by @tpope in `#0002` 

### New Features
* Issue title which has feature-request tag by @fermion in `#0003` 
* Another issue with a feature-request tag by @bensherman in `#0004`

**Comment by MartinX3** [comment]
Yes, the generation should follow the https://www.conventionalcommits.org/en/v1.0.0/#specification enforced by `commitlint` libs and include normal commits.

It makes no sense if I need to open a PR each time I want to rebase my master branch on my development branch after I finished a feature.

**Comment by CaMer0n** [comment]
Would love to hear from someone @github about this. @ettaboyle ?

**Comment by StarSmasher44** [comment]
Is any one there? :'(

**Comment by tiagoliveira9** [comment]
We also have a flow where we cherry pick everything that is going to production to a separate branch, and when releasing, we don't have automatic release notes because the PRs exist on the main branch, not the separate branch which is a bummer.

**Comment by KevinMind** [comment]
We create a tag from master which includes all PRs merged since the last release.

However, sometimes before deploying we need to cherry pick commits that were merged after the initial release tag was cut.

If I create a new release, regardless if I branch from the previous successful release or the unpublished release, the cherry picked commits are not in the change log.

Seems like not only do commits have to be from a PR but also cannot be cherry picked.

**Comment by yuki2006** [comment]
I would like to request a feature that allows selecting commit or pull request titles via a dropdown list in GitHub's auto-generated release notes. This would be particularly useful when creating a pull request that consolidates changes via cherry-picking. It would enhance the flexibility and clarity of the release notes, making them more user-friendly.

**Comment by CreeperIsASpy** [comment]
This is totally an insult for single developers!

**Comment by josegonzalez** [comment]
Longer ticket here: [#31628](https://github.com/orgs/community/discussions/31628)
